docker/Dockerfile.test

```diff
@@ -0,0 +1,20 @@
+FROM screenly/srly-ose-base:latest-x86
+
+ADD requirements/requirements.txt /tmp/requirements.txt
+RUN pip install --no-cache-dir -r /tmp/requirements.txt
+
+ADD requirements/requirements.dev.txt /tmp/requirements.dev.txt
+RUN pip install --no-cache-dir -r /tmp/requirements.dev.txt
+
+RUN pip install --no-cache-dir 'python-vlc==3.0.11115'
```

Comment:
Isn't it better to pull in all requirements from viewer here instead?

Reply:
I'll try using the viewer requirements instead. Thanks!